physics/module_SGSCloud_RadPre.F90

```diff
@@ -17,11 +18,13 @@ end subroutine sgscloud_radpre_finalize
 
 !> \defgroup sgsrad_group GSD sgscloud_radpre_run Module
 !> \ingroup sgscloud_radpre
-!! This interstitial code adds the subgrid clouds to the resolved-scale clouds if there is no resolved-scale clouds in that particular grid box.
+!! This interstitial code adds the subgrid clouds to the resolved-scale clouds 
+!! if there is no resolved-scale clouds in that particular grid box. It can also 
+!! specify a cloud fraction for resolved-scale clouds, using Wu-Randall (1996),
```

Comment:
I guess this should be Xu-Randall and not Wu-Randall? If correct, I'll make the change in the wrapper PR in which I pull your code into the latest version of gsd/develop. No need to change anything.